Client/Assets/Images.xcassets/Shields/shields-share.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "shields-share.pdf",
```

Comment:
We do not use pdfs for assets, change all to pngs

Reply:
Actually using them as vector images saves space on these images but I believe for these icons it will not be a big difference. Making the change so the assets will be consistent.